src/main/java/Duke.java

```diff
@@ -17,34 +17,42 @@ public static void main(String[] args) {
 
         while (!done) {
             line = in.nextLine();
-            if (line.equals("bye")) {
+
+            String[] input = line.split(" ", 2);
+            String command = input[0];
+
+            if (command.equals("bye")) {
                 done = true;
```

Comment:
It might be better to use the variable name "isDone" instead of "done" (if it does not clash with your other booleans)